src/Engine/System/GameSystem.cpp

```diff
@@ -66,7 +66,44 @@ void GameSystem::Player::init() {
     handSpatial = &Scene::addComponent<SpatialComponent>(*gameObject, k_mainHandPosition, headSpatial);
 }
 
+//==============================================================================
+// WAVES
+const Vector<glm::vec3> GameSystem::Wave::k_spawnPoints = {
+    glm::vec3(  0.84, -0.54, -37.80),   // door mid 
+    glm::vec3( 12.70,  0.08,  11.77),   // door back right
+    glm::vec3(-12.08, -0.10,  11.80),   // door back left 
+    glm::vec3(-30.14,  5.55,   8.92),   // seating area back left
+    glm::vec3( 33.81,  5.28,   0.67),   // seating area back right
+    glm::vec3( 11.23,  5.21, -39.39),   // seating area mid back right
+    glm::vec3( -6.42,  5.31, -55.36),   // seating area mid front left 
+};
+int GameSystem::Wave::k_waveNumber = 1;
+float GameSystem::Wave::k_spawnTimer = 0.f;
+float GameSystem::Wave::k_spawnTimerMax = 1.f;
+int GameSystem::Wave::k_enemiesAlive = 0;
+int GameSystem::Wave::k_enemiesInWave = GameSystem::Wave::computeEnemiesInWave();
+float GameSystem::Wave::k_enemyHealth = GameSystem::Wave::computeEnemyHealth();
+float GameSystem::Wave::k_enemySpeed = GameSystem::Wave::computeEnemySpeed();
```

Comment:
the k_ prefix is for constants. So some of these should be const and some I think should loose the k_

Reply:
Replaced with anything?